ajax/libs/rlite-router/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "rlite-router",
+  "version": "1.0.6",
+  "description": "A very tiny, fast client-side router",
+  "main": "rlite.js",
+  "directories": {},
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/chrisdavies/rlite"
+  },
+  "keywords": [
+    "routing"
+  ],
+  "author": "Chris Davies <ferociousturtle@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/chrisdavies/rlite/issues"
+  },
+  "homepage": "https://github.com/chrisdavies/rlite",
+  "npmName": "rlite-router",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "/var/folders/b4/h23tzdm93l563b9ly_5gkyd80000gn/T/tmp-59949w8WBtVhSMvnn/README.md"
```

Comment:
Hmm, did you use `cdnjs-importer` for that? This looks like a bug in cdnjs-importer.